packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -16,6 +16,15 @@ Explore the Administrator Roles API:  [![Run in Postman](https://run.pstmn.io/bu
 
 ## Role Assignment Operations
 
+### Grant Third-Party admin status with the API
```

Comment:
Everything in the API docs is done "with the API" 😄

Reply:
fixed